client/src/seenBy.tsx

```diff
@@ -1,9 +1,16 @@
 import { css } from "@emotion/react";
+import { palette } from "@guardian/source-foundations";
+import { SvgPlus } from "@guardian/source-react-components";
 import React from "react";
 import { LastItemSeenByUser, User } from "../../shared/graphql/graphql";
+import { agateSans } from "../fontNormaliser";
 import { AvatarRoundel } from "./avatarRoundel";
 import { formattedDateTime } from "./util";
 
+const maxSeenByIcons = 2;
```

Comment:
@pradasa having a max seems sensible, 2 seems a too few, IMO

Reply:
I would suggest having 2 and a + sign if we need to display 3 or more avatars. It might get noisy otherwise. 
https://www.figma.com/file/EZZWGDNgPX0s4RUMZTVs4k/%F0%9F%93%8Cchat?node-id=104%3A2978